packages/web3-providers/src/kv/index.ts

```diff
@@ -15,7 +15,7 @@ export class JSON_Storage implements StorageAPI.Storage {
                 {
                     method: 'GET',
                     headers: {
-                        'Content-Type': 'application/json',
+                        'Content-Type': 'text/plain',
```

Comment:
？

Reply:
Interesting.

Reply:
Can you explain why? @lelenei

Reply:
it cant make CORS great again.

Reply:
😨

Reply:
> it cant make CORS great again.

Can you explain in detail? Are we failing on the CORS request? Or do you want to save an OPTIONS request on this?

Reply:
I don't send OPTIONS requests

Reply:
> I don't send OPTIONS requests

Please explain it.

Reply:
Are you trying to avoid preflight requests?

Reply:
yes,
content-type only supports text/plain,multipart/form-data,application/x-www-form-urlencoded